packages/app/src/messages/components/messages/messages.component.tsx

```diff
@@ -316,7 +316,7 @@ const Messages: FunctionComponent<Props> = ({
       )
       if (thread) {
         openThreadDetails(thread)
-      } else {
+      } else if (tmpActiveThread === undefined && thread === undefined) {
```

Comment:
(giving it here, as it is easier to have a thread)

Not sure if this is a scope of this task, but when you type a phone number that is assigned to one of your contacts and then send the message, the user is not redirected to the new thread.

https://user-images.githubusercontent.com/41268731/171599969-a8ba74af-0ad2-4639-b0b6-ca015f49d6fb.mov